types/trace/trace.go

```diff
@@ -212,6 +212,11 @@ type HookedSymbolData struct {
 	ModuleOwner string
 }
 
+type HiddenKernelModule struct {
```

Comment:
since this is on package `trace`, this should be added in a different PR and merged before the reset of the PR.

so one PR with this new type.
another PR with all the rest, and also update the `trace` package in go.mod